src/super_gradients/training/utils/detection_utils.py

```diff
@@ -1164,3 +1168,355 @@ def num_anchors(self) -> int:
 
     def __repr__(self):
         return f"anchors_list: {self.__anchors_list} strides: {self.__strides}"
+
+
+def get_yolox_datadir():
```

Comment:
how is that a util? 
we can add getenv to super_gradients.training.utils.utils.get_param (but not in this PR)

Reply:
removed